<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>177</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="IDPegawai">
      <property name="text">
       <string>ID Pegawai</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Absen">
      <property name="text">
       <string>ID Absen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editIDAbsen">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Tanggal">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="editTanggal"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Jammasuk">
      <property name="text">
       <string>Jam Masuk</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTimeEdit" name="editJamMasuk"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Jamkeluar">
      <property name="text">
       <string>Jam Keluar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTimeEdit" name="editJamKeluar"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Status">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboStatusAbsen">
      <item>
       <property name="text">
        <string>Hadir</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sakit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Izin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alpha</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboIDPegawai"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBersih">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BERSIH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelPegawai">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>IDAbsen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IDPegawai</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jam Masuk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jam Keluar</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
